docs/getting-started.md

```diff
@@ -63,7 +57,7 @@ winget install clarinet
 
 For more information on how to install Clarinet on Windows, please see the [Setting Up Your Clarity Environment (Windows)](https://www.youtube.com/watch?v=r5LY1J5oACs) video walkthrough.
 
-### Install from a pre-built binary
+### Install from pre-built binary
```

Comment:
```suggestion
### Install from a pre-built binary
```
🤷 I guess while we're changing it, why not leave this?

Reply:
Thanks for noticing @MicaiahReid